corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -51,11 +51,16 @@
         $("#archive-form").submit(function() {
             document.getElementById('archive-form-btn').disabled=true;
             document.getElementById('archive-spinner').style.display='inline-block';
-            ga_track_event('Reports', 'Case History', 'Archive this form', {
-                'hitCallback': function () {
+
+            // _.after(2,...) means the callback will only be called after *both* analytics
+            // functions have finished.
+            var analyticsCallback = _.after(2, function() {
```

Comment:
In general, we should either wrap all tracking events in a failsafe function or strictly test these with analytics blockers like [Ghostery](https://chrome.google.com/webstore/detail/mlomiejdfkolichcflejclcbmpeaniij) installed